cmd/govim/config/config.go

```diff
@@ -103,6 +103,17 @@ type Config struct {
 	// results. Zero seconds means unlimited. Examples values: "0s", "100ms"
 	CompletionBudget *string `json:",omitempty"`
 
+	// ExperimentalTempModfile corresponds to the gopls config setting
+	// "tempModfile" which controls whether a temporary modfile is used in place
+	// of the main module's original go.mod file. When enabled, any
+	// user-initiated changes (to .go files) that would otherwise have resulted
+	// in changes to the original go.mod file, e.g. adding an import for a
+	// package whose module is not listed as a requirement, get raised as
+	// diagnostic warnings with suggested fixes which update the go.mod file.
```

Comment:
Hm, in the test case we verify that we don't get any diagnostics.
Is there an issue / CL to refer to until that feature gets implemented? (or should that part of the comment be removed?).

Reply:
Good point. I'll add a link to https://go-review.googlesource.com/c/tools/+/216277